src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -45,7 +46,7 @@ private LostItemPolicy(String id, String name, AutomaticallyChargeableFee declar
     Period itemAgedToLostAfterOverdueInterval, Period patronBilledAfterItemAgedToLostInterval,
     Period recalledItemAgedToLostAfterOverdueInterval,
     Period patronBilledAfterRecalledItemAgedToLostInterval,
-    AutomaticallyChargeableFee ageToLostProcessingFee) {
+    AutomaticallyChargeableFee ageToLostProcessingFee, Period lostItemChargeFeeFine) {
```

Comment:
Period lostItemChargeFeeFine - maybe it makes sense to name it lostItemChargeFeeFineInterval or something like that which will indicate the purpose of this variable